tests/TESTFPC2.MLC

```diff
@@ -99,11 +103,16 @@ WTOPRC_END   EQU   *
          MVC   DHEX,=CL32' '
          LA    R3,L'&VAL
          CALL  CVTTOHEX,(WORK16,DHEX,(R3))
-         WTO   MF=(E,WTOMSG)
+*        WTO   MF=(E,WTOMSG)                             #540
+         PUT   SYSOUT,WTOMSG                             #540
```

Comment:
Another WTO in immediately following IF .. ENDIF. Also, since the WTO is immediately followed by ABEND, should close the DCB.